alarm-frontend/src/Components/AddAlarm.js

```diff
@@ -51,13 +60,28 @@ function AddAlarm() {
                     <option value="AM">Am</option>
                     <option value="PM">Pm</option>
                 </select>
+                <select {...setType}>
+                    <option disabled value="type">
+                        Type
+                    </option>
+                    <option value="Math">Math</option>
+                    <option value="Pokèmon">Pokèmon</option>
+                    <option value="Typing">Typing</option>
```

Comment:
This is fine but if your list is a lot longer, i'd pull the value/label out into an array and do something like 
```
options.map(option => <option value={option}>option</option>)
```